src/registry_schemas/schemas/party.json

```diff
@@ -0,0 +1,113 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/parties",
+    "type": "object",
+    "title": "The Corp Parties Schema",
+    "definitions": {
+        "person": {
+            "type": "object",
+            "properties": {
+                "partyType": {
+                    "type": "string",
+                    "title": "The type of party",
+                    "enum": [
+                        "org",
+                        "person"
+                    ]
+                },
+                "firstName": {
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "lastName": {
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "middleInitial": {
+                    "type": "string",
+                    "maxLength": 30
+                },
+                "orgName": {
+                    "type": "string",
+                    "maxLength": 50
+                },
+                "orgType": {
+                    "type": "string",
+                    "maxLength": 50
+                }
+            },
+            "anyOf": [
+                {
+                  "properties": {
+                    "partyType": { "const": "person" }
+                  },
+                  "required": ["firstName", "lastName"]
+                },
+                {
+                  "properties": {
+                    "partyType": { "const": "org" }
+                  },
+                  "required": ["orgName"]
+                }
+              ],
+            "required": [
+                "partyType"
+            ]
+        },
+        "party": {
+            "type": "object",
+            "properties": {
+                "person": {
+                    "$ref": "#/definitions/person"
+                },
+                "deliveryAddress": {
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
+                },
+                "mailingAddress": {
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
+                },
+                "title": {
+                    "type": "string"
+                },
+                "appointmentDate": {
```

Comment:
What will be the value of this? current date?

Reply:
good question, I added this based on the refactor work...maybe I can just remove it as "required" for now

Reply:
This should be set from the front end, and is the same this as appointmentDate in the current director object

Reply:
The UI does not capture any date values in step 2. We can still pass the value as the current date from UI. But I am not sure whether this is relevant for type Org. So removing 'required' sounds like a good idea to me

Reply:
should be resolved :)